packages/serialization/json/test/common/jsonParseNodeFactory.ts

```diff
@@ -7,4 +7,15 @@ describe("jsonParseNodeFactory", () => {
     const jsonParseNodeFactory = new JsonParseNodeFactory();
     assert.isDefined(jsonParseNodeFactory);
   });
+  it("jsonParseNodeFactory:convertArrayBufferToJson should convert an array to json", async () => {
+    const jsonParseNodeFactory = new JsonParseNodeFactory();
+
+    const expectedJson = '{ "subject": "subject-value" }';
+    const sampleArray = new TextEncoder().encode(expectedJson);
```

Comment:
```suggestion
    const sampleArrayBuffer = new TextEncoder().encode(expectedJson);
```

Reply:
`TextEncoder().encode(expectedJson)` returns a UintArray. Please convert the expectedJson to an arraybuffer as the function being tested takes in `ArrayBuffer`